_minutes/README.md

```diff
@@ -14,11 +14,12 @@ After the end of each meeting, meeting notes are published here.
 
 ## Upcoming meetings
 
-* 2021-08-19 at 11 PM PDT = https://everytimezone.com/?t=611d9f00,708
 * 2021-09-02 at 8 AM PDT = https://everytimezone.com/?t=61301400,384
+* 2021-08-16 at 11 PM PDT = https://everytimezone.com/?t=611d9f00,708
```

Comment:
Should be `09-16` instead of `08-16` and linking to https://everytimezone.com/?t=61428900,708 (provided that we stick to the current schedule).